pkg/library/env/workspaceenv.go

```diff
@@ -80,6 +81,13 @@ func commonEnvironmentVariables(workspaceWithConfig *common.DevWorkspaceWithConf
 			Name:  constants.DevWorkspaceIdleTimeout,
 			Value: workspaceWithConfig.Config.Workspace.IdleTimeout,
 		},
+		{
+			Name:  constants.DISPLAY,
+			Value: ":0",
+		}, {
+			Name:  constants.SSHAskPass,
+			Value: automount.SshAskPassMountPath + automount.SshAskPassScriptFileName,
```

Comment:
In the rest of DWO's codebase, we tend to use `fmt.Sprintf()` instead of using `+` to concatenate strings. In my opinion, it makes it more clear that we are concatenating two strings and not summing two numbers.

Admittedly, `fmt.Sprintf("%s%s", automount.SshAskPassMountPath, automount.SshAskPassScriptFileName)` does look a bit weird however, since we're just concatenating two strings with no extra formatting.

Reply:
done